pypit/arms.py

```diff
@@ -53,10 +60,32 @@ def ARMS(spectrograph, fitstbl, setup_dict):
     basenames = [None]*numsci  # For fluxing at the very end
 
     # Spectrometer class
-    spectro_class = spectro_utils.load_spec_class(spectrograph=spectrograph)
+    if spectrograph is None:
```

Comment:
Eventually, this will move into a better place than here.

e.g. the PipeIt class